### ISSUE TITLE
Pruning Fail

### ISSUE THREAD

**Issue by rongzixuan789** [microsoft/nni#2226]
Hi,

I was trying to test LevelPruner on my network. I used sparsity of 0.8. The pruning has no bug but I found the returning mask has 1 for all the values. Apparently it is not pruning. I am wondering what happened.

**Comment by chicm-ms** [comment]
Hi @rongzixuan789 ,
Would you please paste your network and pruning code? if will be helpful for us to check what happened.

**Comment by rongzixuan789** [comment]
Hi,

Before I posted the code, I am wondering if nni works in google colab. I have installed nni using pip in google colab and run the code without reporting error. But it is not pruning. If it is the problem of using server, I may try the code locally.

**Comment by chicm-ms** [comment]
Hi @rongzixuan789,
Yes it can run on colab, I just wrote a simple example on colab you can check it:
https://colab.research.google.com/drive/1ger68Upwyw_-b3X24phx1tBmneNtCgCs#scrollTo=UFypPjPnhvBf

**Comment by rongzixuan789** [comment]
Hi @chicm-ms 
Thank you for your reply. I think the pruner needs to work in the way that the weights and bias are transferred to the prunermodulewrapper after the model is compressed.

As I have tried to assign the parameters before I compress the model, I got the following.
https://colab.research.google.com/drive/18KD2kR1MdcArgGUyoYqyzl3ZP_hhVDSn

Let's say I have a pretrained and uncompressed model in pth file. When should I load the parameters into the pruner and how should I do it? Thank you
